src/utils/dbUtils.ts

```diff
@@ -170,6 +170,27 @@ const syncedCurrencyCodeMaps = syncedDocument(
 syncedCurrencyCodeMaps.onChange(currencyCodeMaps => {
   logger('Syncing currency code maps with redis cache...')
   for (const key of Object.keys(currencyCodeMaps)) {
+    if (key === 'coingecko') {
```

Comment:
coingecko having lowercase currency codes in currencyCodeMaps is either the result of a current bug or leftover data from an old bug like https://github.com/EdgeApp/edge-rates-server/pull/78/commits/55d0fb2402208def2c26704416aca853fdc5a281

The issue might be resolved by just removing the codes and then this block of code is unnecessary

Reply:
Sounds correct but I think i found a bug that prevented us from ever updating the list. The `coingeckoAssets` func never passed an api key and so would never successfully get a new asset list. I corrected it in a new commit